tests/sqlparser_common.rs

```diff
@@ -3239,14 +3240,14 @@ fn parse_at_timezone() {
             timestamp: Box::new(Expr::Function(Function {
                 name: ObjectName(vec![Ident {
                     value: "FROM_UNIXTIME".to_string(),
-                    quote_style: None
+                    quote_style: None,
                 }]),
                 args: vec![FunctionArg::Unnamed(FunctionArgExpr::Expr(zero.clone()))],
                 over: None,
                 distinct: false,
                 special: false,
             })),
-            time_zone: "UTC-06:00".to_string()
+            time_zone: "UTC-06:00".to_string(),
```

Comment:
I don't mind the `,` everywhere but it sure makes the diff a lot bigger ;)

Reply:
I think this one is from the latest` cargo fmt`